openedx/core/djangoapps/courseware_api/serializers.py

```diff
@@ -107,6 +107,7 @@ class CourseInfoSerializer(serializers.Serializer):  # pylint: disable=abstract-
     show_calculator = serializers.BooleanField()
     original_user_is_staff = serializers.BooleanField()
     can_view_legacy_courseware = serializers.BooleanField()
+    can_access_proctoring_exams = serializers.BooleanField()
```

Comment:
this is a really weird object that says it's about the course but has a bunch of fields about the user

but the new field doesn't make it much weirder